ocs_ci/helpers/osd_resize.py

```diff
@@ -60,6 +60,7 @@ def check_resources_state_post_resize_osd(old_osd_pods, old_osd_pvcs, old_osd_pv
         old_osd_pvs (list): The old osd PV objects before resizing the osd
 
     Raises:
+        StorageSizeNotReflectedException: If the OSD pods failed to restart
         ResourceWrongStatusException: If the following occurs:
```

Comment:
if this raises is not used can we remove this ?

Reply:
I added it in line 79: https://github.com/red-hat-storage/ocs-ci/pull/10641/files#diff-6bcc223cc6becbbfac8729d997ca9a5c7b7cbe57ef19edecdfac9014c6e36749R79.

Reply:
Do you mean to remove the line: 

> 1. The OSD pods failed to reach the status Terminated or to be deleted

Reply:
fixed